internal/app/poweroff/main.go

```diff
@@ -32,6 +35,62 @@ func Main() {
 		log.Fatalf(fmt.Errorf("error while creating machinery client: %w", err).Error())
 	}
 
+	// if we have args, try to parse them in a simple way
+	// combination of args is not supported, args are based on shutdown/poweroff man page
+	if len(args) > 1 {
```

Comment:
we could probably DRY it up a bit, by simply calculating first an action: `shutdown`/`reboot`, and executing it next.

and we could even make it a function: `action(args) -> shutdown/reboot`, and add some unit-tests for it

Reply:
fixed